### ISSUE TITLE
UAC prompt showing

### ISSUE THREAD

**Issue by wolnn** [x0xr00t/Automated-MUlti-UAC-Bypass#33]
Before opening the new PowerShell it shows a UAC prompt all I did was unblock the file and run it

**Comment by x0xr00t** [comment]
Hello @wolnn. 
The stuff you refer to is not a UAC prompt, it's a powershell exec prompt, which ya can bypass by setting execution policy, as described in the readme.MD... Also you can bypass that with type... also this is described in the readme.MD.

I will close this issue for now, if u encounter issues after following the stuff from the readme.MD, then you can make a new ticket, and i will looksie. 

kinds, 
x0xr00t
